app/services/default_values_object.rb

```diff
@@ -0,0 +1,21 @@
+class DefaultValuesObject
```

Comment:
So after reading the post about Feature Envy, to avoid the anti-pattern, we could move back our `set_default_values` method in the `BatchController`, which is so far the only place where the `@form` variable gets modified to account for default values. If I understand correctly, from the refactoring post, the rule is to place or define the new feature where the data gets changed right?